types/material-ui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for material-ui v0.16.0
+// Type definitions for material-ui v0.16.1
```

Comment:
Yes but I mean - should the _types_ version be 0.17.1 to match or will @types/material-ui just always get the latest one. I guess I don't know the details - I am just going off of what I am seeing in other libraries per my comment above. I don't want people thinking that this PR adds whatever other types may have been introduced in 0.17.1. It's your call, guys.